src/renderer/modals/certificate/certificate.tsx

```diff
@@ -5,14 +5,27 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import {Certificate} from 'electron/renderer';
 
-import {MODAL_CANCEL, MODAL_RESULT, RETRIEVE_MODAL_INFO} from 'common/communication';
-
-import SelectCertificateModal from './certificateModal';
+import {DARK_MODE_CHANGE, GET_DARK_MODE, MODAL_CANCEL, MODAL_RESULT, RETRIEVE_MODAL_INFO} from 'common/communication';
 
 import 'bootstrap/dist/css/bootstrap.min.css';
 import 'renderer/css/modals.css';
 import 'renderer/css/components/CertificateModal.css';
 
+import darkStyles from 'renderer/css/lazy/modals-dark.lazy.css';
+
+import SelectCertificateModal from './certificateModal';
+
+window.addEventListener('message', async (event) => {
+    if (event.data.type === DARK_MODE_CHANGE) {
+        if (event.data.data) {
+            darkStyles.use();
```

Comment:
I didn't know about this functionality from `style-loader`. That's really cool actually.

Would it perhaps be possible to not repeat this event handler in multiple files? Perhaps there could be a helper in common for importing the correct CSS files for theming as a one-liner instead of repeating this each time? 0/5 though.

Reply:
I'm actually not sure if it works that way or not, but maybe the import tree would pick it up if it was in a helper. I'll give it a shot.